bundle-workflow/src/test_workflow/utils/bundle_manifest_provider.py

```diff
@@ -0,0 +1,35 @@
+# SPDX-License-Identifier: Apache-2.0
+#
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+#
+# Modifications Copyright OpenSearch Contributors. See
+# GitHub history for details.
+
+import os
+
+from aws.s3_bucket import S3Bucket
+from manifests.bundle_manifest import BundleManifest
+
+
+class BundleManifestProvider:
```

Comment:
open to moving these methods to bundle_manifest.py if that makes more sense

Reply:
Yes.